packages/cli/test/unit/index.test.ts

```diff
@@ -1,14 +1,6 @@
 import os from 'node:os';
-import {
-  describe,
-  expect,
-  it,
-  vi,
-  beforeEach,
-  afterEach,
-  beforeAll,
-  afterAll,
-} from 'vitest';
+import { describe, expect, it, vi, beforeAll, afterAll } from 'vitest';
+import 'ci-info';
```

Comment:
Why is this import here?

Reply:
Ah, just leftover from when the test was going to live there. Lint didn't catch it because it could be used for side effect.